src/Grid.tsx

```diff
@@ -8,7 +8,13 @@ import {
 import type { GridRowDetailsReactRendererProps } from './renderers/grid.js';
 import { useModelRenderer } from './renderers/useModelRenderer.js';
 
-export * from './generated/Grid.js';
+// "@vaadin/grid/vaadin-grid.js" has additional re-exports from "grid-column.js"
+// for historical reasons. This exports "GridColumn" web component, which gets
+// suggested in React context, see https://github.com/vaadin/react-components/issues/68
+// Fix: use re-exports from raw "src/vaadin-grid.js" as a workaround, until
+// the re-export is removed.
+export * from '@vaadin/grid/src/vaadin-grid.js';
```

Comment:
What about the `import './theme/lumo/vaadin-grid.js'` from `@vaadin/grid/vaadin-grid.js`?

Reply:
We still have at least one line in `generated/Grid.ts` that brings it in:
```typescript
import { Grid as GridElement, type GridEventMap as _GridEventMap, } from "@vaadin/grid/vaadin-grid.js";
```